include/circt/Dialect/FIRRTL/FIRRTLAttributes.h

```diff
@@ -33,22 +33,35 @@ namespace direction {
 
 /// Return an output direction if \p isOutput is true, otherwise return an
 /// input direction.
-inline Direction get(bool isOutput) { return (Direction)isOutput; }
+static inline Direction get(bool isOutput) { return (Direction)isOutput; }
+
+/// Convert from Direction to bool.  The opposite of get;
+static inline bool unGet(Direction dir) { return (bool)dir; }
 
 /// Flip a port direction.
 Direction flip(Direction direction);
 
-inline StringRef toString(Direction direction) {
+static inline StringRef toString(Direction direction) {
   return direction == Direction::In ? "in" : "out";
 }
 
+static inline StringRef toString(bool direction) {
+  return toString(get(direction));
+}
+
+/// Return a \p DenseBoolArrayAttr containing the packed representation of an array
+/// of directions.
+mlir::DenseBoolArrayAttr packAttribute(MLIRContext *context,
+                                       ArrayRef<Direction> directions);
+
 /// Return a \p IntegerAttr containing the packed representation of an array
```

Comment:
```suggestion
/// Return a \p DenseBoolArrayAttr containing the packed representation of an array
```
The comment should be updated.